src/gui/systray.h

```diff
@@ -42,6 +42,7 @@ class AccessManagerFactory : public QQmlNetworkAccessManagerFactory
 bool canOsXSendUserNotification();
 void sendOsXUserNotification(const QString &title, const QString &message);
 void setTrayWindowLevelAndVisibleOnAllSpaces(QWindow *window);
+double statusBarThickness();
```

Comment:
@claucambra Not sure about `Mac-specific` code, but, can you make it a const member-function?